### PR TITLE
M

### LINKED ISSUE #38: git push -u origin Master
git push -u origin Master

_Originally posted by @Ottapettavan in https://github.com/Ottapettavan/SHAH-MWOL/issues/35#issuecomment-2018335362_

### LINKED ISSUE #40: Fix code scanning alert - Incomplete regular expression for hostnames
Here is a simple footnote[^1].

A footnote can also have multiple lines[^2].

[^1]: My reference.
[^2]: To add line breaks within a footnote, prefix new lines with 2 spaces.
  This is a second line.

Tracking issue for:
- [x] https://github.com/Ottapettavan/SHAH-MWOL/security/code-scanning/9

### LINKED ISSUE #42: > ### Discussed in #28
> ### Discussed in #28
> Originally posted by **Ottapettavan** March 24, 2024
> 
> _This discussion was created from the release [Node.json](https://github.com/Ottapettavan/SHAH-MWOL/releases/tag/)._
#38 #33 #34 #25

_Originally posted by @Ottapettavan in https://github.com/Ottapettavan/SHAH-MWOL/issues/29#issuecomment-2018346935_

### LINKED ISSUE #48: Fix code scanning alert - Incomplete URL substring sanitization
Tracking issue for:
- [x] https://github.com/Ottapettavan/SHAH-MWOL/security/code-scanning/7

### LINKED ISSUE #52: Fix code scanning alert - Incomplete URL substring sanitization
Tracking issue for:
- [x] https://github.com/Ottapettavan/SHAH-MWOL/security/code-scanning/6

### PR DESCRIPTION
**Pull request by Ottapettavan**
😌😌

### CODE REVIEW

**Comment by Ottapettavan** [comment]
> 😌😌

Duplicate of #44 #31

File: .github/workflows/codacy.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        uses: codacy/codacy-analysis-cli-action@d840f886c4bd4edc059706d09c6a1586111c540b
+        with:
+          # Check https://github.com/codacy/codacy-analysis-cli#project-token to get your project token from your Codacy repository
+          # You can also omit the token and run the tools that support default configurations
+          project-token: ${{ secrets.CODACY_PROJECT_TOKEN }}
+          verbose: true
+          output: results.sarif
+          format: sarif
+          # Adjust severity of non-security issues
+          gh-code-scanning-compat: true
```
Comment (Ottapettavan) [inline .github/workflows/codacy.yml]:
```suggestion
          gh-code-scanning-compat: true
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
```
Comment (ramshad-sha) [inline .github/ISSUE_TEMPLATE/bug_report.md:2]:
```suggestion
name: Bug report
```

File: .github/ISSUE_TEMPLATE/custom.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (ramshad-sha) [inline .github/ISSUE_TEMPLATE/custom.md:1]: @Ottapettavan

**Pull request merged**